src/Features/Core/Portable/AddParameter/AddParameterService.cs

```diff
@@ -130,6 +132,14 @@ public bool HasCascadingDeclarations(IMethodSymbol method)
                         insertionIndex++;
                     }
 
+                    if (offerRename)
+                    {
+                        var paramToken = syntaxFacts.GetIdentifierOfParameter(parameterDeclaration);
+                        var newParamToken = paramToken.WithAdditionalAnnotations(RenameAnnotation.Create());
+                        parameterDeclaration = parameterDeclaration.ReplaceToken(paramToken, newParamToken);
+                        //parameterDeclaration = parameterDeclaration.WithAdditionalAnnotations(RenameAnnotation.Create());
```

Comment:
remove.

Reply:
this is in a loop.  so it looks like multiple parametrs could offer rename.  taht seems busted.
i also don't really understand why this is needed here.  we'd off rename *not* on the parameter (which coudl be distantly inserted) but on teh replacement at the location the user is at.